docs/src/use/migrate-to-10.0.0.md

```diff
@@ -38,3 +40,19 @@ ESLint is officially dropping support for these versions of Node.js starting wit
 **To address:** Make sure you upgrade to at least Node.js v20.19.0 when using ESLint v10. One important thing to double check is the Node.js version supported by your editor when using ESLint via editor integrations. If you are unable to upgrade, we recommend continuing to use ESLint v9 until you are able to upgrade Node.js.
 
 **Related issue(s):** [#19969](https://github.com/eslint/eslint/issues/19969)
+
+## <a name="config-lookup-from-file"></a> New configuration file lookup algorithm
+
+In ESLint v9, the alternate config lookup behavior could be enabled with the `v10_config_lookup_from_file` feature flag. This behavior made ESLint locate `eslint.config.*` by starting from the directory of each linted file and searching up towards the filesystem root. In ESLint v10, this behavior is now the default and the `v10_config_lookup_from_file` flag has been removed. Attempting to use this flag will now result in an error.
+
+**To address:**
+
+- Remove any usage of the flag in your setup:
+    - CLI: remove `--flag v10_config_lookup_from_file`.
+    - Environment: remove `v10_config_lookup_from_file` from `ESLINT_FLAGS`.
+    - API: remove `"v10_config_lookup_from_file"` from the `flags` array passed to `new ESLint()` or `new Linter()`.
+- If you relied on the previous (cwd-based) lookup behavior, either:
+    - Run ESLint from the directory containing the desired config file, or
```

Comment:
I'm not sure how this helps achieve the previous behavior, since the lookup will still be from the files being linted. Can you clarify?